integration/commands_administration_test.go

```diff
@@ -915,6 +915,49 @@ func TestCommandsAdministrationServerStatusMetrics(t *testing.T) {
 	}
 }
 
+func TestCommandsAdministrationServerStatusFreeMonitoring(t *testing.T) {
+	// this test shouldn't be run in parallel, because it requires specific state of the field which would be modify by the other tests.
```

Comment:
```suggestion
	// this test shouldn't be run in parallel, because it requires a specific state of the field which would be modified by the other tests.
```